docs/topics/pipeline.md

```diff
@@ -21,13 +20,6 @@ GZIP_COMPRESSION_RATE=1
 ```
 SELENIUM_LOG_LEVEL=trace
 ```
-#### Filter test-runs by file
```

Comment:
already described in testing.md and this is not a recommended way of using this.